.env.example

```diff
@@ -63,3 +63,8 @@ CLIENT_SECRET_GITHUB_LOGIN=
 
 CLIENT_ID_GITLAB_LOGIN=
 CLIENT_SECRET_GITLAB_LOGIN=
+
+CAPTCHA_SECRET=
+CAPTCHA_ENABLED=
```

Comment:
nit: can't we determine if captcha is enabled by checking if CAPTCHA_SECRET is set?